### PR TITLE
EXP-102 -  Esconder a aba equipe existente na pilot para clientes link.me

### PR DESCRIPTION
**Pull request by diegodsgarcia**
## Contexto
Esconde Tab Equipe quando uma company logada na Pilot for tipo `payment_link`

## Issues linkadas
- [x] [EXP-102 - Esconder a aba equipe existente na pilot para clientes link.me](https://mundipagg.atlassian.net/secure/RapidBoard.jspa?rapidView=185&projectKey=EXP&modal=detail&selectedIssue=EXP-102)

### Company COM payment_link
image

### Company SEM payment_link
image

## Como testar?
- Entre em uma company na pilot **SEM** o tipo `payment_link_app` e veja se aparece a tab Equipes na página de Configurações
- Entre em uma company na pilot **COM** o tipo `payment_link_app` e veja se desaparece a tab Equipes na página de Configurações

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 077109a8589ab38ada088f5fdc899e9d0a1d9747
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 077109a8589ab38ada088f5fdc899e9d0a1d9747
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 077109a8589ab38ada088f5fdc899e9d0a1d9747
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 077109a8589ab38ada088f5fdc899e9d0a1d9747
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19b61c64eb07d51dddb322ab3a1c058af289ecfa
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19b61c64eb07d51dddb322ab3a1c058af289ecfa
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19b61c64eb07d51dddb322ab3a1c058af289ecfa
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19b61c64eb07d51dddb322ab3a1c058af289ecfa
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by allangrds** [comment]
- [x] Entre em uma company na pilot SEM o tipo payment_link_app e veja se aparece a tab Equipes na página de Configurações
- [x] Entre em uma company na pilot COM o tipo payment_link_app e veja se desaparece a tab Equipes na página de Configurações

File: packages/pilot/src/containers/Settings/Company/index.js [hunk 1]
```diff
@@ -281,6 +285,7 @@ CompanySettings.propTypes = {
     name: PropTypes.string,
     role: PropTypes.string,
   })).isRequired,
+  type: PropTypes.string.isRequired,
```
Comment (vagnervst) [inline packages/pilot/src/containers/Settings/Company/index.js:288]: Ao fazer o refresh da página, o type vem `null` aqui pois ainda estamos carregando a company. Acho que seria melhor remover o `isRequired`

File: packages/pilot/src/containers/Settings/Company/index.js [hunk 2]
```diff
@@ -80,7 +81,10 @@ class CompanySettings extends Component {
           >
             <TabItem text={t('pages.settings.company.tab.general')} />
             <TabItem text={t('pages.settings.company.tab.products')} />
-            <TabItem text={t('pages.settings.company.tab.team')} />
+            {type !== 'payment_link_app'
+              ? <TabItem text={t('pages.settings.company.tab.team')} />
+              : <></>
```
Comment (alexandrejuk) [inline packages/pilot/src/containers/Settings/Company/index.js:86]: aqui poderia retornar null

Comment (diegodsgarcia) [reply packages/pilot/src/containers/Settings/Company/index.js:86]: Eu tentei com null, porém quando coloco qualquer coisa que retorne q não seja um Node HTML, da erro no componente. Por isso fiz dessa forma.

Comment (alexandrejuk) [reply packages/pilot/src/containers/Settings/Company/index.js:86]: E se tentarmos assim 

```js
  {type !== 'payment_link_app' && (<TabItem text={t('pages.settings.company.tab.team')} />)}
```

Comment (diegodsgarcia) [reply packages/pilot/src/containers/Settings/Company/index.js:86]: Infelizmente não funciona. Pois assim também é retornado no final `undefined` no child do componente :( 
image

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b65b31fdfb1af83219d534860e9b7fb306ed0442
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b65b31fdfb1af83219d534860e9b7fb306ed0442
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b65b31fdfb1af83219d534860e9b7fb306ed0442
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b65b31fdfb1af83219d534860e9b7fb306ed0442
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8b51f737850e38fe27d3cd9aa60d26038dd849d4
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8b51f737850e38fe27d3cd9aa60d26038dd849d4
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8b51f737850e38fe27d3cd9aa60d26038dd849d4
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8b51f737850e38fe27d3cd9aa60d26038dd849d4
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Pull request merged**